events/apigw.go

```diff
@@ -42,6 +42,32 @@ type APIGatewayProxyRequestContext struct {
 	APIID         string                    `json:"apiId"` // The API Gateway rest API Id
 }
 
+// APIGatewayHTTPRequest contains data coming from the new HTTP API Gateway
+type APIGatewayHTTPRequest struct {
+	Version               string                       `json:"version"`
+	RouteKey              string                       `json:"routeKey"`
+	RawPath               string                       `json:"rawPath"`
+	RawQueryString        string                       `json:"rawQueryString"`
+	Headers               map[string]string            `json:"headers"`
+	QueryStringParameters map[string]string            `json:"queryStringParameters"`
+	PathParameters        map[string]string            `json:"pathParameters"`
+	RequestContext        APIGatewayHTTPRequestContext `json:"requestContext"`
+	Body                  string                       `json:"body"`
+	IsBase64Encoded       bool                         `json:"isBase64Encoded,omitempty"`
+}
+
+// APIGatewayHTTPRequestContext contains the information to identify the AWS account and resources invoking the
+// Lambda function. It also includes Cognito identity information for the caller.
+type APIGatewayHTTPRequestContext struct {
+	RouteKey   string                    `json:"routeKey"`
```

Comment:
Fields appear to be missing in this too.

Reply:
Added other fields